src/sentry/sentry_apps/services/app_request/model.py

```diff
@@ -14,6 +15,11 @@ class RpcSentryAppRequest(RpcModel):
     webhook_url: str
     organization_id: int
     event_type: str
+    error_id: str | None = None
+    project_id: int | None = None
+    request_body: str | None = None
+    request_headers: Mapping[str, str] | None = None
```

Comment:
Is it necessary to capture the header data here? I'm not sure what we'd necessarily find useful to surface/if we have anything we'd prefer not exposing via the UI here, though I see that we're already exposing it atm.

Reply:
ehhhh so the important info would be the timestamp & signature. The signature is encrypted with the client secret, would people be able to decrypt via that info ? Albeit the people who have access to this resource would only be the sentry app integrator.

Usefulness wise, I think the timestamp could be p. useful in debugging cases, but the rest 🤷‍♀️ Would more info than not, would be preferred ?

Reply:
going to leave the header data here so that we can reuse the [RequestSerializer](https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_apps/api/serializers/request.py) for the new endpoint

Reply:
Q: We're missing a few fields from the RequestSerializer on the model, should we be adding those?

Reply:
which ones?

Reply:
if i understand this correctly, those get added by the response serializer, so they're not in the sentry app request model